libs/ledger-live-common/src/families/cosmos/preloadedData.mock.ts

```diff
@@ -5,1013 +5,1412 @@ const LEDGER_VALIDATOR_ADDRESS = cryptoFactory("cosmos").ledgerValidator;
+      commission: 0,
       estimatedYearlyRewardsRate: 0,
     },
     {
       validatorAddress: "cosmosvaloper1rwh0cxa72d3yle3r4l8gd7vyphrmjy2kpe4x72",
-      tokens: 5232494848288,
-      votingPower: 5232494848288 / (0.7202 * 254624972017526),
-      name: "SparkPool",
-      commission: 0.04,
+      name: "pe4x72",
+      tokens: 241429222763,
+      votingPower: 0,
+      commission: 0.2,
+      estimatedYearlyRewardsRate: 0,
+    },
+    {
+      validatorAddress: "cosmosvaloper1rj6324uq904z5zr96zg6ew9qjyau9u6h5nflg6",
+      name: "Don Cryptonium",
+      tokens: 82593677494,
+      votingPower: 0,
+      commission: 0.05,
       estimatedYearlyRewardsRate: 0,
     },
     {
       validatorAddress: "cosmosvaloper1rcp29q3hpd246n6qak7jluqep4v006cdsc2kkl",
-      tokens: 243735228951,
-      votingPower: 243735228951 / (0.7202 * 254624972017526),
-      name: "2nd only to Certus One in GoS: in3s.com",
-      commission: 1.0,
+      name: "in3s.com",
+      tokens: 152636184646,
+      votingPower: 0,
+      commission: 0.01,
+      estimatedYearlyRewardsRate: 0,
+    },
+    {
+      validatorAddress: "cosmosvaloper1yfnaup5wa3vdzx3wx9auhvzl85saqj37tqxqnu",
+      name: "\bMoonlet",
+      tokens: 84781248304,
+      votingPower: 0,
+      commission: 0.05,
+      estimatedYearlyRewardsRate: 0,
+    },
+    {
+      validatorAddress: "cosmosvaloper1yw5s259jkcg0jzmh7sce29uk0lqqw2ump7578p",
+      name: "CEX.IO ❤️🇺🇦",
+      tokens: 134566485728,
+      votingPower: 0,
+      commission: 0.05,
       estimatedYearlyRewardsRate: 0,
     },
     {
       validatorAddress: "cosmosvaloper1y0us8xvsvfvqkk9c6nt5cfyu5au5tww2ztve7q",
-      tokens: 51573500874,
-      votingPower: 51573500874 / (0.7202 * 254624972017526),
       name: "Swiss Staking",
-      commission: 0.0,
+      tokens: 1892339743010,
+      votingPower: 0,
+      commission: 0.01,
       estimatedYearlyRewardsRate: 0,
     },
     {
-      validatorAddress: "cosmosvaloper19yy989ka5usws6gsd8vl94y7l6ssgdwsrnscjc",
-      tokens: 500001550260,
-      votingPower: 500001550260 / (0.7202 * 254624972017526),
-      name: "OKEx Pool",
-      commission: 0.1,
+      validatorAddress: "cosmosvaloper1yh089p0cre4nhpdqw35uzde5amg3qzexkeggdn",
+      name: "High Stakes 🇨🇭",
+      tokens: 79996206173,
+      votingPower: 0,
+      commission: 0.05,
       estimatedYearlyRewardsRate: 0,
     },
     {
-      validatorAddress: "cosmosvaloper1998928nfs697ep5d825y5jah0nq9zrtd00yyj7",
-      tokens: 106330127188,
-      votingPower: 106330127188 / (0.7202 * 254624972017526),
-      name: "HLT",
+      validatorAddress: "cosmosvaloper19rmlxkqjt950fxl849l49x0me56u9tkd5n5j62",
+      name: "Post Road",
+      tokens: 1006285498547,
+      votingPower: 0,
       commission: 0.1,
       estimatedYearlyRewardsRate: 0,
     },
     {
-      validatorAddress: "cosmosvaloper199mlc7fr6ll5t54w7tts7f4s0cvnqgc59nmuxf",
-      tokens: 572547259593,
-      votingPower: 572547259593 / (0.7202 * 254624972017526),
```

Comment:
Why did we remove votingPower everywhere ? I don't really know the implications as a mock tho... @hzheng-ledger 

Rest is fine :)

Reply:
This is the response that we are getting from back-end.
The question is, what is the real implication? (It could be a bk bug)